src/pykiso/test_coordinator/test_execution.py

```diff
@@ -252,6 +252,9 @@ def execute(
         log.exception("Keyboard Interrupt detected")
         exit_code = ExitCode.ONE_OR_MORE_TESTS_RAISED_UNEXPECTED_EXCEPTION
     except Exception:
-        log.exception(f'Issue detected in the test-suite: {config["test_suite_list"]}!')
+        log.exception(
```

Comment:
I don't get this, the principle of log.exception is to log the message at error level AND to print out the exception stack, so even without this change you'd see all of the errors